config/dependencies/quickstart.yaml

```diff
@@ -43,6 +43,8 @@ spec:
       containers:
         - command:
             - etcd
+            - --data-dir
```

Comment:
Just a small thing but might be nice to add a short comment here explaining why it's done

Reply:
How about this 148a7a4

```YAML
        - command:
            - etcd
            - --data-dir  # use data directory under /tmp for read/write access by non-root user on OpenShift
            - /tmp/etcd.data
```